user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Template_Editor.md

```diff
@@ -0,0 +1,338 @@
+
+- To modify the **template settings**, make sure no layers are selected. In the *Settings* pane, you can specify a number of pixels to which the width and/or height of the template will be fixed.
+
+- To modify the **layer settings**, make sure a layer is selected. In the *Settings* pane, you can adjust the following layer settings in three collapsible sections:
+
+  - [Dimensions](#specifying-layer-dimensions)
+
+  - [Properties](#specifying-layer-properties)
+
+  - [Conditional cases](#adding-conditional-cases-to-a-layer)
+
+### Specifying layer dimensions
+
+The table below describes the dimension settings available for template layers<!--RN 37476-->.
+
+| Icon | Setting | Description |
+|--|--|--|
+| ![Width](~/user-guide/images/Dimensions_Width.png) | Width | Specify a number of pixels to which the width of the layer will be fixed. This value should be lower than or equal to the width of the overall template. |
+| ![Height](~/user-guide/images/Dimensions_Height.png) | Height | Specify a number of pixels to which the height of the layer will be fixed. This value should be lower than or equal to the height of the overall template. |
+| ![Top](~/user-guide/images/Dimensions_Top.png) | Top | Specify a number of pixels to which the top of the layer will be fixed. This value should be lower than the height of the overall template. |
+| ![Bottom](~/user-guide/images/Dimensions_Bottom.png) | Bottom | Specify a number of pixels to which the bottom of the layer will be fixed. This value should be lower than the height of the overall template. |
+| ![Left](~/user-guide/images/Dimensions_Left.png) | Left | Specify a number of pixels to which the left of the layer will be fixed. This value should be lower than the width of the overall template. |
+| ![Right](~/user-guide/images/Dimensions_Right.png) | Right | Specify a number of pixels to which the right of the layer will be fixed. This value should be lower than the width of the overall template. |
+
+> [!NOTE]
+>
+> - In the preview, when you use the drag-and-drop functionality to move a layer, or when you resize a layer by dragging the frame edges, the dimension settings in the *Settings* pane will automatically adjust.
+> - You can change the unit to percentages by clicking *px* next to the dimension values.
+> - You can use the lock functionality to enhance the security of your design, locking one or multiple dimension settings of a layer by clicking the ![*Lock*](~/user-guide/images/Dimensions_Lock.png) button next to the chosen setting. Once locked, any attempt to override this specified dimension will be restricted, including moving or resizing that layer in the preview and resizing the component in the dashboard or low-code app edit mode (e.g. changing the size of a table column). The locked dimensions will ensure the template remains in its designated position. If you cannot lock a dimension setting because of the locking of other settings, it will appear grayed out.
```

Comment:
Might be good to mention that this can be used to make sure the template looks like they want when it is variable in width or height. In the table for example, the use of the locks is really importent because you can rescale the width of any column

Reply:
Feedback implemented in commit [c5a531e](https://github.com/SkylineCommunications/dataminer-docs/pull/2328/commits/c5a531ea38eb524b55d42150f16caa60890a4dc5).